### ISSUE TITLE
Add a device execution view

### ISSUE THREAD

**Issue by email2vimalraj** [openstf/stf#394]
## What's the problem (or question)?
I'm an author of https://github.com/email2vimalraj/appium-stf-example. There is no problem, however it is an ask for the new feature. As for the automation, after executing the script, we would like to watch the execution. So it would be great if we can add an execution view where it shows real time streaming of test execution.

## What should have happened?
I triggered an execution using appium and opened the STF dashboard which shown a Devices page. I clicked on the device where the execution is happening, because I was expecting it will show me the execution. But the moment I clicked on it, it released the device from me causing the automation failure.

## Do you have an idea for a solution?
Idea way could be add an execution view where user can view the live execution with no controls (Ex: Perfecto's device dashboard).
Another suggestion would be, we should not immediately allocate / release device when the user clicks on the particular device on the Devices view. Rather, it should be a button to select / release. Because, I sometimes mistakenly clicked on the device causing allocation / releasing.

## Help us understand your issue by providing context.
This ask is better to have and help for automation.

## Please tell us details about your environment.
* Are you using `stf local` or a [full deployment](doc/DEPLOYMENT.md): `stf local`
* Version used (git hash or `stf -V`):  `v2.0.1'
* Environment name and version (e.g. Chrome 39, node.js 5.4): `v6.2.1`
* Operating System and version: `Mac OSx 10.11.5`

**Comment by vbanthia** [comment]
> I triggered an execution using appium and opened the STF dashboard which shown a Devices page. I clicked on the device where the execution is happening, because I was expecting it will show me the execution. But the moment I clicked on it, it released the device from me causing the automation failure.

I don't think it is releasing device until you are clicking "Stop Using" button. 

To view the execution,
-> Go to device list
-> Click device control url from "Product" column as shown in this demo video, https://github.com/openstf/stf-appium-example/blob/master/doc/demo.gif

**Comment by vbanthia** [comment]
Also you must be logged in from same user whose access token you are using.

**Comment by email2vimalraj** [comment]
@vbanthia : Yes that solves the purpose. But still I'm looking a separate execution view with nothing other than the screen streaming. This will avoid people playing around with un-necessary interactions while the execution is on.

And when the execution is completed, I'm getting this:

<img width="1249" alt="screen shot 2016-08-10 at 8 28 49 am" src="https://cloud.githubusercontent.com/assets/1214686/17540567/c4f36742-5ed4-11e6-9375-247ff12700dd.png">

**Comment by gunta** [comment]
@email2vimalraj are you looking to even prevent touches to the screen?

By the way if you press `SHIFT + CLICK` you can see only the view, is this something similar to what you want?

**Comment by email2vimalraj** [comment]
@gunta : Yes something similar to SHIFT+CLICK and prevent touches.
